src/routes/auth.routes.js

```diff
@@ -1,12 +1,15 @@
-import { verifySignUp } from "../middlewares";
+import checkUsername from "../middlewares/verifysignup";
 import controller from "../controllers/auth.controller";
```

Comment:
Name the controller in a self-explanatory way like e.g: **UserController**  don't leave it raw like that it doesn't have any meaning.